src/Listener/CodeCoverageListener.php

```diff
@@ -76,6 +83,7 @@ public function __construct(ConsoleIO $io, CodeCoverage $coverage, array $report
         ];
 
         $this->skipCoverage = $skipCoverage;
+        $this->coversUtil = new CoversAnnotationUtil(new Registry());
```

Comment:
Maybe add the `if (!class_exists('SebastianBergmann\CodeUnit\InterfaceUnit')) { ... }` here, so that no unnecessary code is loaded.

And a few lines down:
```php
        if (is_null($this->coversUtil)) {
            $this->coverage->stop();
            return;
        }
```